### DISCUSSION TITLE
想识别为拼音，但WER一直为1正常吗？

### DISCUSSION THREAD

**Discussion by gezimonkey** [PaddlePaddle/PaddleSpeech#593] [Q&A]
我尝试使用DeepSpeech训练自己的数据来识别带声调的拼音(例如：pu3 tong1 hua4)
数据大约有500小时采样率为16K的wav，质量一般，我进行了降噪和vad处理
训练时设置力lang_model_path为空，其他参数未动

epoch>60后，loss在0.6-1.2之间震荡
test结果全部为< u n k >
wer等于1

我想知道问题出在哪？

**Comment by zh794390558** [comment]
可以看下train的结果，如果还是unk的话再看vocab有没有问题。
